src/pages/module-reference/module-wishlist-graph-ql.md

```diff
@@ -24,24 +24,24 @@ Before installing this module, note that the Magento_WishlistGraphQl is dependen
 - Magento_Sales
 - Magento_Store
 
-For information about enabling or disabling a module, see [Enable or disable modules](https://devdocs.magento.com/guides/v2.4/install-gde/install/cli/install-cli-subcommands-enable.html).
+For information about enabling or disabling a module, see [Enable or disable modules](https://experienceleague.adobe.com/docs/commerce-operations/installation-guide/tutorials/manage-modules.html).
 
 ## Extensibility
 
-Extension developers can interact with the Magento_WishlistGraphQl module. For more information about the Magento extension mechanism, see [Magento plug-ins](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/plugins.html).
+Extension developers can interact with the Magento_WishlistGraphQl module. For more information about the Magento extension mechanism, see [Magento plug-ins](https://developer.adobe.com/commerce/php/development/components/plugins/).
 
-[The Magento dependency injection mechanism](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/depend-inj.html) enables you to override the functionality of the Magento_WishlistGraphQl module.
+[The Magento dependency injection mechanism](https://developer.adobe.com/commerce/php/development/components/dependency-injection/) enables you to override the functionality of the Magento_WishlistGraphQl module.
 
 ## Additional information
 
 For more information about the Magento_WishlistGraphQl queries and mutations see below:
 
 ### Queries
 
-- [`wishlist`](https://devdocs.magento.com/guides/v2.4/graphql/queries/wishlist.html)
+- [`wishlist`](https://developer.adobe.com/commerce/webapi/graphql/schema/wishlist/queries/wishlist/)
 
 ### Mutations
 
-- [`addProductsToWishlist`](https://devdocs.magento.com/guides/v2.4/graphql/mutations/add-products-to-wishlist.html)
-- [`removeProductsFromWishlist`](https://devdocs.magento.com/guides/v2.4/graphql/mutations/remove-products-from-wishlist.html)
-- [`updateProductsInWishlist`](https://devdocs.magento.com/guides/v2.4/graphql/mutations/update-products-in-wishlist.html)
+- [`addProductsToWishlist`](https://developer.adobe.com/commerce/webapi/graphql/schema/wishlist/mutations/add-products/)
+- [`removeProductsFromWishlist`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/remove-products-from-wishlist/)
```

Comment:
```suggestion
- [`removeProductsFromWishlist`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/remove-products/)
```

Reply:
https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/remove-products/ is still 404

Reply:
https://developer.adobe.com/commerce/webapi/graphql/schema/wishlist/mutations/remove-products/